archlinux.pkr.hcl

```
@@ -185,7 +185,7 @@ build {
   provisioner "shell" {
     inline = [
       "arch-chroot /mnt/archinstall pacman -S --noconfirm virtualbox-guest-utils",
-      "arch-chroot /mnt/archinstall modprobe -a vboxguest vboxsf vboxvideo" 
+      // "arch-chroot /mnt/archinstall modprobe -a vboxguest vboxsf vboxvideo",     modprobe fails
```

I think PR is okay, but uncommenting this  leads guest-utils not working. I don't have x64 machine at the moment to test why it was not working now.
```suggestion
      "arch-chroot /mnt/archinstall modprobe -a vboxguest vboxsf vboxvideo",     // modprobe fails sometimes?
```